README.md

```diff
@@ -5,13 +5,104 @@ Group a list of signals into one hash table signal. The output signal will conta
 
 If *one_value* is True, the Signal attributes will be just a single matching value instead of a list of all matching values. If multiple matches, then the last signal processed will be the value used.
 
+If *group_by* is used, a Signal attribute (key) with the default (pre-assigned) name *group* will have the value of *group_by* in the output signal.
+
+**Example:**
```

Comment:
I think "Examples" might be better at the very end of the doc, especially because it's such a long section. What do you think?